texthero/visualization.py

```diff
@@ -306,3 +308,123 @@ def top_words(s: TextSeries, normalize=False) -> pd.Series:
         .explode()  # one word for each line
         .value_counts(normalize=normalize)
     )
+
+
+def visualize_describe(df, text_col_name="text", labels_col_name="topic"):
```

Comment:
`labels_col_name="topic"` not sure it make sense to have as default value "topic". Moreover, there might be multiple non-text columns.

Going rither, as we are interested in giving users information regarding text-columns, I would limit this function to handle text-columns. I.e replace `text_col_name ` with `text_cols_names` (or similar) and accept both a list of string or just a string (or only a list of string, as you wish.). I suggest to drop `labels_col_name` but I'm open to discussion and to see your point of view

Reply:
see comment below